cmd/podman/pods/rm.go

```diff
@@ -72,23 +73,44 @@ func init() {
 }
 
 func rm(cmd *cobra.Command, args []string) error {
-	ids, err := specgenutil.ReadPodIDFiles(rmOptions.PodIDFiles)
-	if err != nil {
-		return err
-	}
-	args = append(args, ids...)
+	var errs utils.OutputErrors
+
 	if cmd.Flag("time").Changed {
 		if !rmOptions.Force {
 			return errors.New("--force option must be specified to use the --time option")
 		}
 		rmOptions.Timeout = &stopTimeout
 	}
-	return removePods(args, rmOptions.PodRmOptions, true)
+
+	// First parse all ID files to return potential read errors before
+	// starting to remove pods.
+	idFileMap := make(map[string]string, len(rmOptions.PodIDFiles)) // file -> ID
+	for _, idFile := range rmOptions.PodIDFiles {
+		id, err := specgenutil.ReadPodIDFile(idFile)
+		if err != nil {
+			return err
```

Comment:
We cannot stop for this. Bad or missing ID files cannot interrupt removal. We can certainly log errors, but we have to keep going

Reply:
That seems frustrating for users - a file having the wrong permissions should not prevent me removing a pod or container. I am in favor of changing, because what we're doing now seems silly.

Reply:
I can change but it's not really something this PR did.